draft-ietf-ppm-dap.md

```diff
@@ -939,12 +939,12 @@ The validation checks are as follows.
    is marked as invalid with the error `batch-collected`. This prevents
    additional reports from being aggregated after its batch has already
    been collected.
-2. Check if the report has already been aggregated. If this check fails,
-   the input share is marked as invalid with the error `report-replayed`.
-   This is the case if the report was used in a previous aggregate request
-   and is therefore a replay. An aggregator may also choose to mark an
-   input share as invalid with the  error `report-dropped` under the conditions
-   prescribed in {{anti-replay}}.
+2. Check if the report has already been aggregated with this aggregation
```

Comment:
Note: the reference to an "aggregation parameter" in this section is new. However, I think it's OK: this section is referenced from the Leader Initialization & the Helper Initialization sections. The Leader Initialization section already assumes an aggregation parameter, and the Helper Initialization section explicitly receives an aggregation parameter from the leader's request. So I think this reference is well-formed.